amqp-protocol-binding.md

```diff
@@ -151,13 +127,25 @@ directly to the CloudEvents `datacontenttype` attribute.
 
 #### 3.1.2. Event Data Encoding
 
-The [`data`](#22-data) byte-sequence is used as the AMQP
+Event data is assumed to contain opaque application data that is
+encoded as declared by the `datacontenttype` attribute.
+
+An application is free to hold the information in any in-memory representation
+of its choosing, but as it is transposed into AMQP as defined in this
+specification, the assumption is that the event data is made available as a
+sequence of bytes. The byte sequence is used as the AMQP
 [application-data][data] section.
 
+Example:
+
+If the declared `datacontenttype` is `application/json;charset=utf-8`, the
+expectation is that the event data is made available as [UTF-8][rfc3629] encoded
+JSON text for use in AMQP.
+
 #### 3.1.3. Metadata Headers
 
-All [CloudEvents][ce] attributes with exception of `datacontenttype` and `data`
-MUST be individually mapped to and from the AMQP
+All [CloudEvents][ce] attributes with exception of `datacontenttype` MUST be
+individually mapped to and from the AMQP
 [application-properties][app-properties] section, with exceptions noted below.
```

Comment:
Strike ", with exceptions noted below", we already mentioned datacontentype and there aren't any more exceptions.